lib/ansible/modules/system/gather_facts.py

```diff
@@ -25,8 +25,8 @@
 
 DOCUMENTATION = '''
 ---
-module: setup
-version_added: historical
+module: gather_facts
+version_added: 2.4
```

Comment:
The value for `version_added` should remain `historical` since we're only renaming the module.